plugins/github-issues/src/utils/generateTestIssue.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { faker } from '@faker-js/faker';
```

Comment:
Faker is unfortunately very big package(9mb unpacked) that we want to avoid having as regular dependency. If you really want to continue using it you probably have to inline the generateTestIssue in your dev index.ts in order to be able to place it under `devDependencies` again. Ideally you can store a fixture(json file) with some nice looking test data that you load and use for the dev server and by doing that can completely remove the dependency.

Reply:
Makes sense. I'll make the change today.

Reply:
Done here: [f6fb510](https://github.com/backstage/backstage/pull/13059/commits/f6fb510ec4ab9162bd9b0d6de5a07493da358e7d)